tests/testthat/test-po_compile.R

```diff
@@ -14,3 +14,18 @@ test_that("get_po_metadata() returns 0 rows if no .po fles", {
   meta <- get_po_metadata(temp)
   expect_equal(nrow(meta), 0)
 })
+
+test_that("po_compile() can handle UTF-8 msgstr", { temp <-
+  local_test_package(
```

Comment:
```suggestion
test_that("po_compile() can handle UTF-8 msgstr", { 
  temp <- local_test_package(
```

Reply:
ah thanks. this weekend I learned fat-fingering Ctrl+F in nano completely botches R files 😅

Reply:
I spent at least a minute wondering if this was some genuine google syntax 😛